### ISSUE TITLE
About loss_keypoints_3d

### ISSUE THREAD

**Issue by SecretGuardian** [nkolot/GraphCMR#13]
Hi I used dataset option 'itw' to train a new model by using default params. The 'loss_keypoints_3d' and 'loss_keypoints_3d_smpl' are always 0. Thus the results are worse than the pretrained model. 

Do you have any other recommendations to provide the 3D pose information? Since human3.6M never replies to my email. And I checked the gt_keypoints_3d = input_batch['pose_3d'] and it shows:

tensor([[[0., 0., 0., 0.],      
         [0., 0., 0., 0.], ...

Can you please give the joints name order of the 3D joints in training process and explain what these four 0.s stand for? Thanks a lot.

**Comment by nkolot** [comment]
All the "In The Wild" datasets do not have 3D pose data available. In these cases our dataloader passes in an array of zeros (all x,y,z and confidence are 0). So the loss (that is weighted by the confidence) will always be 0. What you are observing is the intented behavior.

**Comment by SecretGuardian** [comment]
Got it, thank you for explaining :)
